modelskill/comparison/_collection_plotter.py

```diff
@@ -213,13 +216,14 @@ def kde(self, ax=None, figsize=None, title=None, **kwargs) -> Axes:
             df_model = df[df.model == model]
             df_model.mod_val.plot.kde(ax=ax, label=model, **kwargs)
 
-        # TODO use unit_text from the first comparer
-        # TODO make sure they are conistent
-        # then it should be a property of the collection, not only the comparer
-        ax.set_xlabel(f"{self.cc[0].unit_text}")
+        ax.set_xlabel(f"{self.cc.unit_text}")
 
-        default_title = f"KDE plot for {', '.join(cmp.name for cmp in self.cc)}"
-        ax.set_title(title or default_title)
+        title = (
+            f"KDE plot for {', '.join(cmp.name for cmp in self.cc)}"
```

Comment:
Limit the length of default title. If longer than e.g. 100 characters then set to txt[:80] + "..." or similar.